torchtext/experimental/datasets/language_modeling.py

```diff
@@ -76,7 +77,9 @@ def _setup_datasets(dataset_name, tokenizer, root, vocab, data_select, single_li
             raw_train, = raw.DATASETS[dataset_name](root=root, data_select=('train',), year=year, language=language)
         else:
             raw_train, = raw.DATASETS[dataset_name](root=root, data_select=('train',))
+        logging.info('Building Vocab based on train data')
         vocab = build_vocab(raw_train, tokenizer)
+    logging.info('Vocab has {} entries'.format(len(vocab)))
```

Comment:
Instead of building the string before passing it to the logger object, a common way is to let the logger object construct the string.

```python
info('Vocab has %s entries', len(vocab))
```

Reply:
added